examples/SampleApp/package.json

```diff
@@ -45,7 +45,7 @@
     "react-native-image-resizer": "1.4.5",
     "react-native-markdown-package": "1.8.2",
     "react-native-quick-sqlite": "8.0.2",
-    "react-native-reanimated": "^3.1.0",
+    "react-native-reanimated": "~3.1.0",
```

Comment:
Done because of [this](https://github.com/GetStream/stream-chat-react-native/pull/2136/files#r1217632200).